### PR TITLE
Add documentation yaml to collect documentation for internal DOCs too

### PR DESCRIPTION
**Pull request by itaiche**
### 🔦 Summary
Adding the documentation to internal repo
...

### ✅ Checklist

### CODE REVIEW

File: documentation.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  docs:
+    - file: README.md
+      title: Getting Started
+    - file: MIGRATION.md
+      title: Migration
+    - file: docs/usage/LAYOUTING.md
+      title: Layouting
+    - file: docs/usage/MOBILE_WEB_GUIDELINES.md
+      title: Mobile
+    - file: usage-with-create-stylable-app.md
```
Comment (TalZa1810) [inline documentation.yaml:17]:
```suggestion
    - file: docs/usage/usage-with-create-stylable-app.md
```

File: documentation.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - file: docs/usage/COMPONENTS_DRIVERS.md
+      title: Testing/Component Drivers
+    - file: docs/usage/testing_pitfalls.md
+      title: Testing/Pitfalls
+    - file: docs/internal/DEPRECATION_GUIDE.md
+      title: Deprecation
+    - file: docs/FAQ/README.md
+      title: FAQ/Questions
+    - file: docs/FAQ/TYPES.MD
+      title: FAQ/TypeScriptw
```
Comment (TalZa1810) [inline documentation.yaml:32]:
```suggestion
      title: FAQ/TypeScript
```

**Review by TalZa1810 — CHANGES_REQUESTED** [review]
Few typos that I've found

**Comment by TalZa1810** [comment]
Also, make sure that [you sign your commits ](https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md#sign-commits)so that the CI will run

File: documentation.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+apiDoc:
+  title: Wix Style React
+  description: Our official Dashboard© and Back-Office© design system
+  category: fe-guild
```
[the remark this answers is not in the document]
Comment (itaiche) [reply documentation.yaml:4]: Nope, this is aligned with our internal docs path.
It's not `fed-guild` :)

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6425

**Review by TalZa1810 — APPROVED** [review]

**Pull request merged**